tests/components/netatmo/test_media_source.py

```diff
@@ -64,6 +64,9 @@ async def test_async_browse_media(hass: HomeAssistant) -> None:
     # Test successful listing
     media = await async_browse_media(hass, f"{URI_SCHEME}{DOMAIN}/events/")
 
+    # Test successful media children is not None
+    assert media.children
```

Comment:
I think media.children can be None if  camera_id is not None and all cameras with the specific event cannot find a media url

Reply:
hmmm maybe you are right. but maybe we ignore that 🫣☺️